### PR TITLE
New Security side nav updates 

### LINKED ISSUE #3694: [Meta] Update Security UI side nav
Related PR: https://github.com/elastic/kibana/pull/159185
Public issue: https://github.com/elastic/kibana/issues/157847

Major changes: 
- Former **Manage -> Rules** is moving to **Rules**. 
- **Manage -> Cloud Posture Benchmarks** is moving under **Rules -> Benchmark rules**. 
- **Manage -> Shared Exception lists** is moving under **Rules -> Shared Exception lists**. 

### SideNav package

- Styles updated according to the design
- New `categories` prop for the main panel links.
- New types of categories for the secondary panel links:
  - Title: Groups links under a category with a title label.
  - Separator: Groups links adding space with surrounding links
  - Accordion: Groups links in a EUI collapsible accordion (not used yet, will be needed for Serverless) 
- The `footerItems` component prop has been removed in favor of the `position?: 'top' | 'bottom' // default 'top'` item attribute.
- New `iconType` optional item attribute to display the icon next to the SideNav link. 
- All links texts now wrap instead of truncate.
- Links descriptions are no longer used.
- Secondary panel size updated, width reduced from `340px` to `270px`.

**Before**

**After** 
image

```[tasklist]
### Topics to review for updates:
- [ ] [UI Overview](https://www.elastic.co/guide/en/security/current/es-ui-overview.html)
```

### PR DESCRIPTION
**Pull request by natasha-moore-elastic**
- Resolves #3694 
- Preview: [Elastic Security UI](https://security-docs_3785.docs-preview.app.elstc.co/guide/en/security/master/es-ui-overview.html)

#### Updated paths:
- [x] **Manage -> Rules** to **Rules -> Detection rules (SIEM**)
- [x] **Manage -> CSP Benchmarks** to **Rules -> Benchmark rules**
- [x] **Manage -> Shared Exception lists** to **Rules -> Shared exception lists**

#### Updated images/gifs that display the left nav:
- [Elastic Security UI](https://www.elastic.co/guide/en/security/current/es-ui-overview.html):
    - [x] [images/nav-overview.gif](https://www.elastic.co/guide/en/security/current/images/nav-overview.gif)
    - [x] [manage-pg.png](https://www.elastic.co/guide/en/security/current/manage-pg.png)
- [AI Assistant](https://www.elastic.co/guide/en/security/current/security-assistant.html):
    - [x] [images/assistant.gif](https://www.elastic.co/guide/en/security/current/images/assistant.gif) (if possible, @joepeeples to re-do the gif)

#### Updated content:
- [Elastic Security UI](https://www.elastic.co/guide/en/security/current/es-ui-overview.html):
    - [x] Under **Manage**, removed Rules, Share Exception Lists, and CSP Benchmarks bullet points
    - [x] Swapped **Cases** and **Timelines** sections
    - [x] Swapped **Explore** and **Intelligence** sections
    - [x] Between **Explore** and **Get started**, added a new **Rules** section

### CODE REVIEW

**Comment by joepeeples** [comment]
Added the updated AI Assistant GIF in https://github.com/elastic/security-docs/pull/3785/commits/e7efb4ece2c3da6a8605fe90c2e9514d07aad241.

File: docs/getting-started/security-ui.asciidoc [hunk 1]
```diff
@@ -137,12 +145,24 @@ image::images/network-ui.png[Shows the Network page]
 image::images/users/users-page.png[Shows the Users page]
 
 [float]
-=== Intelligence
+=== Rules
 
-The Intelligence section contains the Indicators page, which collects data from enabled threat intelligence feeds and provides a centralized view of indicators of compromise (IoCs). Refer to <<indicators-of-compromise>> to learn more.
+Expand this section to view the following pages:
 
+* <<rules-ui-create, *Rules*>>: Create and manage rules to monitor suspicious events.
```
Comment (banderror) [inline docs/getting-started/security-ui.asciidoc:152]: The *Rules* -> *Detection rules (SIEM)* nav link leads to the Rule Management page. Shouldn't we link to [`guide/en/security/master/rules-ui-management.html`](https://www.elastic.co/guide/en/security/master/rules-ui-management.html) here accordingly?

**Review by banderror — APPROVED** [review]
Thanks for including me @natasha-moore-elastic, everything LGTM. I left one minor comment.

File: docs/getting-started/security-ui.asciidoc (hunk 1 above)
Comment (natasha-moore-elastic) [reply docs/getting-started/security-ui.asciidoc]: That makes sense to me, thanks @banderror!
Resolved in [d6bf835](https://github.com/elastic/security-docs/pull/3785/commits/d6bf835092f07d2f7d7e2b6a115ba9d8ffbd069e).

**Review by semd — APPROVED** [review]

File: docs/getting-started/security-ui.asciidoc [hunk 2]
```diff
@@ -98,6 +98,14 @@ Compare your Kubernetes infrastructure against a variety of security benchmarks.
 [role="screenshot"]
 image::cloud-native-security/images/findings-page.png[The Findings page]
 
+[float]
+=== Cases
+
+Open and track security issues. Refer to <<cases-overview>> to learn more.
+
+[role="screenshot"]
+image::cases/images/cases-home-page.png[Shows the Cases page]
```
Comment (joepeeples) [inline docs/getting-started/security-ui.asciidoc:107]:
```suggestion
image::cases/images/cases-home-page.png[Cases page]
```

**Review by joepeeples — COMMENTED** [review]
One tiny nit, LGTM!

**Pull request merged**